openapi.yaml

```diff
@@ -1278,6 +1420,34 @@ components:
         vertexType:
           $ref: '#/components/schemas/VertexType'
 
+    ReachablePlace:
+      description: Place reachable by One-to-All
+      type: object
+      properties:
+        place:
+          $ref: "#/components/schemas/Place"
+          description: Place reached by One-to-All
+        duration:
+          type: integer
+          description: Total travel duration
+        connectionCount:
```

Comment:
what's a "connection"? Does this require more explanation?
Even the code has more explanation. Why not have the same name?
(either both times `k` or both times `connectionCount` but not 2 names for the same concept if we can avoid it)